toolkit/tools/internal/pkggraph/pkggraph.go

```diff
@@ -1434,3 +1577,78 @@ func findAllRPMS(rpmsToFind []string) (foundAllRpms bool, missingRpms []string)
 
 	return
 }
+
+// findAllRPMS returns true if all RPMs requested are found in PMC/repo.
+//  This is used to check if all RPMs are available in PMC/repo before starting a build.
+//  needs repo-list, manifest files rpmrepocloner
+//	Also returns a list of all missing files
+func findAllRPMSinPMC(rpmsToFind map[*pkgjson.PackageVer]bool, outDir, tmpDir, workerTar string, existingRpmsDir string, existingToolchainRpmsDir string, usePreviewRepo bool, repoFiles []string) (foundAllRpms bool, missingRpms []string) {
```

Comment:
This feels like a candidate for inclusion in rpmrepocloner as a public function, thoughts? Again I might be able to use this to optimize my delta fetcher work.

Reply:
All functions in rpmrepocloner are wrappers around various tdnf commands. This function re-uses existing functions in cloner, doesn't introduce any new tdnf command. So it need not be included in the rpmrepocloner. It can be made public and can be used by included pkggraph